.npmignore

```diff
@@ -1,8 +1,5 @@
-coverage
-dist
 node_modules
 test
 .editorconfig
 .eslintrc
 .eslintignore
-.travis.yml
```

Comment:
Since you're using `files` in `package.json`, I don't know if you need this file at all?

Reply:
Ah yeah, I think you're right. I can clean this up.